src/Tests/Sfa.Tl.ResultsAndCertification.Web.UnitTests/Controllers/AdminDashboardControllerTests/ChangeIndustryPlacementGet/When_Called_With_Valid_Data.cs

```diff
@@ -0,0 +1,98 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using NSubstitute;
+using Sfa.Tl.ResultsAndCertification.Common.Helpers;
+using Sfa.Tl.ResultsAndCertification.Web.Content.AdminDashboard;
+using Sfa.Tl.ResultsAndCertification.Web.ViewModel.AdminDashboard;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Sfa.Tl.ResultsAndCertification.Web.UnitTests.Controllers.AdminDashboardControllerTests.ChangeIndustryPlacementGet
+{
+    public class When_Called_With_Valid_Data : TestSetup
+    {
+        public int PathwayId { get; set; }
```

Comment:
Lines 14to 16 can be removed. Declared in parent class

Reply:
Done